tests/rptest/tests/cluster_bootstrap_test.py

```diff
@@ -110,7 +106,8 @@ def __init__(self, test_context):
     def setUp(self):
         # NOTE: `rpk redpanda admin brokers list` requires versions v22.1.x and
         # above.
-        self.installer.install(self.redpanda.nodes, self.OLDVER)
+        self.OLDVER, self.OLDVER_STR = self.installer.install(
```

Comment:
nit: all-caps names should only be used for constants.

Reply:
fixed